src/Uno.UI/Microsoft/UI/Xaml/Controls/Expander/Expander.Properties.cs

```diff
@@ -45,7 +45,7 @@ public DataTemplate HeaderTemplate
 				nameof(HeaderTemplate),
 				typeof(DataTemplate),
 				typeof(Expander),
-				new PropertyMetadata(null));
+				new FrameworkPropertyMetadata(null, FrameworkPropertyMetadataOptions.ValueDoesNotInheritDataContext));
```

Comment:
Maybe I missed some of the conversation, but why did you add this option?

Reply:
@jeromelaban This was requested in https://github.com/unoplatform/uno/pull/6630#discussion_r685865886

Reply:
Thanks!